packages/buckets/src/api/index.ts

```diff
@@ -120,31 +132,49 @@ export enum PathAccessRole {
   PATH_ACCESS_ROLE_ADMIN = 3,
 }
 
-export type MetadataObject = {
+export type BuckMetadata = {
   roles: Map<string, PathAccessRole>
   updatedAt: number
 }
+/**
+ * @deprecated
+ */
+export type MetadataObject = BuckMetadata
 
 /**
  * A bucket path item response
  */
-export type PathItemObject = {
+export type PathItem = {
   cid: string
   name: string
   path: string
   size: number
   isDir: boolean
-  items: Array<PathItemObject>
+  items: Array<PathItem>
   count: number
-  metadata?: MetadataObject
+  metadata?: BuckMetadata
 }
+/**
+ * @deprecated
+ */
+export type PathItemObject = PathItem
 
 /**
  * A bucket list path response
  */
-export type PathObject = {
-  item?: PathItemObject
-  root?: RootObject
+export type Path = {
+  item?: PathItem
+  root?: Root
+}
+/**
+ * @deprecated
+ */
+export type PathObject = Path
+
+export const CHUNK_SIZE = 32768
+
+export function* genChunks(value: Uint8Array, n: number) {
```

Comment:
Naive chunk generator that simple chunks things into smaller bits, but includes everything if it can.

Reply:
should `n` be renamed since it is really `s` or `size` not number?